src/qibolab/designs/mixed.py

```diff
@@ -0,0 +1,61 @@
+from dataclasses import dataclass
+
+from qibo.config import log, raise_error
+
+from qibolab.designs.basic import BasicInstrumentDesign
+
+
+@dataclass
+class MixedInstrumentDesign(BasicInstrumentDesign):
```

Comment:
I find `Mixed` funny, shall we use `Mixer` instead?

Reply:
> @stavros11 thank you very much. This indeed simplifies **a lot** the hardware specifics from the library, in particular the new `create_tii_qw25_gold` function looks very good.

Thank you for the review. I agree, and after we cleanup the `platform.setup` moving the lab specifics to the same function, in principle this could be used as a mechanism for external users to plug their own platforms:
```py
def create_my_own_platform():
    # define the `Platform` using an `InstrumentDesign`
    return platform


platform = create_my_own_platform()
```
without touching any qibolab code. The user can also decide if he wants to parse the lab specifics from a runcard or hardcode them inside the function.

The only open question from my side is whether these two instrument designs are sufficient to accommodate all TII instruments.

> If this approach is stable we should merge as soon as possible.

At least for QM, all scripts and calibrations I tested with #259 work with this branch too.